src/Shared/PackageManagers/NPMProjectManager.cs

```diff
@@ -158,7 +158,7 @@ public override async Task<IEnumerable<string>> EnumerateVersions(PackageURL pur
                 string? packageName = purl.Namespace != null ? $"@{purl.Namespace}/{purl.Name}" : purl.Name;
                 HttpClient httpClient = CreateHttpClient();
 
-                string? content = await GetHttpStringCache(httpClient, $"{ENV_NPM_API_ENDPOINT}/{packageName}");
+                string? content = await GetHttpStringCache(httpClient, $"{ENV_NPM_API_ENDPOINT}/{packageName}", false);
```

Comment:
This should respect the UseCache parameter. See later comments - enumerate versions should match the other apis and have an optional usecache parameter.  The default pattern has been to set these to true when not present, and it should then be passed in here.  This will mean that you can get your preferred non-cache behavior but the default behavior does not change.

Reply:
Yeah, I can revert it there. I was just having issues with the cached responses for metadata. I dont think EnumerateVersions was the culprit though.